src/components/Accordion/Accordion.jsx

```diff
@@ -0,0 +1,48 @@
+import { Disclosure } from "@headlessui/react";
+import clsx from "clsx";
+import React, { Children } from "react";
+import { SlideDown } from "../Animation/SlideDown";
+import { ChevronRightIcon } from "../../icons/ChevronRightIcon";
+
+export const Accordion = ({ children }) => {
+    const childrenArray = Children.toArray(children);
+    return (
+        <div>
```

Comment:
`Disclosure` itself should have a `div`. Remove this div if you don't need it. Simplifies the markup, and if the end-user (seller app, admin etc) need an extra div, they can add it

Reply:
Removed